.github/workflows/container-autoupdate.yml

```diff
@@ -0,0 +1,34 @@
+name: Build, test and push container
+on:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+jobs:
+  push-container:
+    name: Push container to Quay.io
```

Comment:
Nitpick: "Refresh container", as it also includes the build step. And let's not repeat the destination too much.

Reply:
KK, will fix that.